include/courtier/GWebsocketConsumerT.hpp

```diff
@@ -1508,6 +1513,445 @@ class GWebsocketConsumerT
 	 //-------------------------------------------------------------------------
 };
 
+
+// Add just another consumer
+
+template<typename processable_type>
+class GWebsocketConsumerPT
```

Comment:
What is the rationale behind this? Is this an alternative to the "old" GWebSocketConsumerT or meant to be used alternatively?